helm/values-dev.yaml

```
@@ -0,0 +1,17 @@
+image:
+  repository: selcdacr.azurecr.io/selfcareexternalapi
```

here should go the repo name without "-" (it's configured in gitops project)
```suggestion
  repository: selcdacr.azurecr.io/selfcareexternalapibackend
```